tagr/storage/aws.py

```diff
@@ -63,3 +63,19 @@ def dump_pickle(self, pickle_object, proj, experiment, tag, filename):
         self.S3.Object(proj, "{}/{}/{}.pkl".format(experiment, tag, filename)).put(
             Body=pickle_object
         )
+
+    def list(self, proj, experiment, tag):
```

Comment:
I see we have a method named `list()` in `artifacts.py`. lets make this naming explicit. rename to private function